modules/reactor/doc/content/source/meshgenerators/TriangulatedMeshGenerator.md

```diff
@@ -0,0 +1,33 @@
+# TriangulatedMeshGenerator
+
+!syntax description /Mesh/TriangulatedMeshGenerator
+
+## Overview
+
+The `TriangulatedMeshGenerator` was created to connect the functionality of the [poly2tri](https://github.com/jhasse/poly2tri) 2D triangulation library to the MOOSE mesh generation system. Since the motivation of this effort was to mesh the region between a reactor fuel core and the core periphery (typically a cylinder), it was designed to create a circular triangulation outer boundary and use the outer boundary of an existing meshgenerator to define the boundary of a single hole region which the triangulation library will exclude. The outer circle properties are specified with the [!param](/Mesh/TriangulatedMeshGenerator/outer_circle_radius) and [!param](/Mesh/TriangulatedMeshGenerator/outer_circle_num_segments) input parameters.
```

Comment:
I'm not a fan of the name. We need to reserve `TriangulatedMeshGenerator` for a general mesh generator in the framework.

Reply:
Strongly agree.  I'm not sure what to call this one, though.  TriangulatedAnnulusGenerator?  Except the region is only an annulus topologically, not geometrically.

Reply:
We have another MG capability that is called PeripheralMeshGenerator, maybe this can be PeripheralTriangleMeshGenerator